README.md

```diff
@@ -79,127 +90,3 @@ Are available under _spack/etc/spack_. Their structure is:
 	<li>-repos.yaml (specifies where to find the only mch packages that are stored in spack-mch repository)</li>
 </ul>
 
-
-# <img src="https://cdn.rawgit.com/spack/spack/develop/share/spack/logo/spack-logo.svg" width="64" valign="middle" alt="Spack"/> Spack
```

Comment:
This looks like a copy paste from the spack repo. It will be obsolete as soon as the repo moves... so I propose to just keep the link  to spack project at the top